src/lib/prisma-session-store.spec.ts

```diff
@@ -1,20 +1,15 @@
-// tslint:disable: no-default-import
 // tslint:disable: no-duplicate-string
 
-import { PrismaClient } from '@prisma/client';
+import type { IOptions, IPrisma } from '../@types';
+import { createPrismaMock } from '../mocks';
 
-import type { IOptions } from '../@types';
-import { createPrismaMock, MockStore } from '../mocks';
-
-import prismaSessionStore from './prisma-session-store';
+import { PrismaSessionStore } from './prisma-session-store';
 import { createExpiration, range, sleep } from './utils/testing';
 
 jest.mock('./utils/defer', () => ({
   defer: (fn: Function, ...args: unknown[]) => fn(...args),
 }));
 
-const PrismaSessionStore = prismaSessionStore({ Store: MockStore });
-
 /**
  * Creates a new `PrismaSessionStore` and prisma mock
  * @param options any additional options for the store
```

Comment:
Yes, this would be options that alter the behavior from the default settings.
🤔  is there a better way to word this?

Reply:
I'm probably over-thinking things, but it seems conceivable that "additional" could be interpreted in two ways...

1) Perhaps an `express-session`'s store requires a set of default options. (Reviewing their README, I'm not seeing a set of required _options_ - only _methods_ that must be supported - but I do wonder if there is nonetheless a set of required or conventionally offered options). Regardless, this store for Prisma _does_ extend the set of options provided by _roccomuso's_ session store. So potentially, "additional" could be construed as referring to one or more of the options that are _only_ available from _this_ library (e.g. `logger`, `loggerLevel`), not available from (for example) roccomuso's session store.

2) "additional" could also (potentially?) be construed to refer to any options you want to _explicitly_ set values for, in order to over-ride whatever default values are provided.

See what you think - I could very well be over-thinking this... (I'm a UI designer by trade, and spend a lot of time imagining how people could mis-interpret things!)

Reply:
Don't get me wrong, I definitely do how this could be misleading.
I will take the day to think about how to better word this and update it by the end of the day.

Reply:
Updated.